src/c++/library/common.h

```diff
@@ -364,6 +371,8 @@ class InferInput {
   IOType io_type_;
   std::string shm_name_;
   size_t shm_offset_;
+
+  bool binary_data_{true};
```

Comment:
when I see binary_data, I assume it has binary data, not a bool.  `is_binary_data_` perhaps?  Same for the function BinaryData()

Reply:
Trying to keep parity with python client binary data bool, it's named binary_data, not is_binary_data:

https://github.com/triton-inference-server/client/blob/5bb0da1579204982e00f19a1ac7a5b8b4a46c7e0/src/python/library/tritonclient/http/_infer_input.py#L92

Reply:
Also similar to description in extension: 
https://github.com/triton-inference-server/server/blob/main/docs/protocol/extension_binary_data.md

Reply:
ok, lets keep consistency.